### ISSUE TITLE
[suggestion] Improved Sounding Infusion Enchantment

### ISSUE THREAD

**Issue by Orlikys** [Azanor/thaumcraft-suggestions#202]
Each level of Sounding enchantment adds color information on ore types:
1 level - all ores are white (no addiotional information)
2 level - Coal- Dark Grey, Iron - Brown
3 level - Gold - Yellow, Red Stone - Red, Lazurite - Dark Blue
4 level - Diamond - Light Blue, Emerald - Green

**Comment by Ulysses2** [comment]
Just to clarify, are you asking for Sounding to be nerfed? Because the TC6 version already color-codes ores.

**Comment by Orlikys** [comment]
No, I didn't know thet. Then I apologize, the topic can be deleted..
